apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -822,10 +822,13 @@ static void addCareTeamQualification(
     }
 
     Coding coding = createCoding(rootResource, ccwVariable, code.get());
-    CodeableConcept concept = new CodeableConcept();
-    concept.addCoding(coding);
-
-    careTeam.setQualification(concept);
+    // Only add the qualification if it doesn't already exist
+    if (careTeam.getQualification().getCoding().stream()
```

Comment:
What's the reason for this change? Are we sure `careTeam.getQualification()` can't be null here?

Reply:
Ah I see. Thanks!